subsys/bluetooth/audio/pacs.c

```diff
@@ -608,3 +649,28 @@ int bt_pacs_location_changed(enum bt_audio_dir dir)
 	return 0;
 }
 #endif /* CONFIG_BT_PAC_SNK_LOC || CONFIG_BT_PAC_SRC_LOC */
+
+static void available_contexts_notify(struct k_work *work)
+{
+	struct bt_pacs_context context;
+	int err;
+
+	err = available_contexts_get(NULL, &context);
+	if (err) {
+		BT_DBG("get_available_contexts returned %d, won't notify", err);
+		return;
+	}
+
+	err = bt_gatt_notify_uuid(NULL, BT_UUID_PACS_AVAILABLE_CONTEXT, pacs_svc.attrs,
+				  &context, sizeof(context));
+	if (err != 0) {
+		BT_WARN("Available Audio Contexts notify failed: %d", err);
+	}
+}
+
+static K_WORK_DELAYABLE_DEFINE(available_contexts_work, available_contexts_notify);
+
+int bt_pacs_available_contexts_changed(void)
+{
+	return k_work_reschedule(&available_contexts_work, PAC_NOTIFY_TIMEOUT);
```

Comment:
k_work_reschedule() may return positive value which isn't an error,  is that intended?   bt_pacs_available_contexts_changed() caller seems to assume non-zero is an error

Reply:
Yes, you have right. Just to explain myself I made this like that, because the timeout here is != `K_NO_WAIT`. In that case `k_work_reschedule` won't return an error according to documentation.
However, I agree with you that it's not best choice as API might change or `PAC_NOTIFY_TIMEOUT` may be changed to `K_NO_WAIT`.

Reply:
@sjanc You are absolute right. I must overlook this part:
```
1 if delay not K_NO_WAIT and work has been scheduled
```